### PR TITLE
Issue192 interactive charts

### PR DESCRIPTION
**Pull request by wd15**
Adds interactivity to the vega free energy plots including panning, zooming and add/remove data.

### CODE REVIEW

File: _data/charts/benchmark_upload.json [hunk 1]
```diff
@@ -1,207 +1,207 @@
...
     {
-      "domain": false,
+      "domain": false, 
       "encode": {
         "labels": {
           "update": {
             "font": {
               "value": "Lato"
-            },
+            },
```
Comment (tkphd) [inline _data/charts/benchmark_upload.json]: Are the trailing spaces necessary or desirable?

File: _data/charts/code_upload.json [hunk 2]
```diff
@@ -1,191 +1,191 @@
 {
   "axes": [
     {
-      "domain": false,
+      "domain": false,
```
Comment (tkphd) [inline _data/charts/code_upload.json]: Are the trailing spaces necessary or desirable?

File: _data/charts/benchmark_upload.json [hunk 3]
```diff
@@ -1,207 +1,207 @@
...
           "update": "{}"
         }
-      ],
+      ], 
       "value": {}
     }
-  ],
+  ], 
   "width": 230
 }
```
Comment (tkphd) [inline _data/charts/benchmark_upload.json]: POSIX-compliant files end with a newline character.

File: _data/charts/code_upload.json [hunk 4]
```diff
@@ -1,191 +1,191 @@
...
           "update": "{}"
         }
-      ],
+      ], 
       "value": {}
     }
-  ],
+  ], 
   "width": 230
 }
```
Comment (tkphd) [inline _data/charts/code_upload.json]: POSIX-compliant files end with a newline character.

**Comment by wd15** [comment]
@tkphd, I fixed the white space issues. I was building the JSON using Python 2 tools instead of Python 3. It changed the JSON formatting. Please merge it now. Thanks.

**Comment by tkphd** [comment]
LGTM. Merging.

**Pull request merged**